helm-chart/renku-core/templates/deployment.yaml

```diff
@@ -40,18 +46,52 @@ spec:
             - name: RENKU_SVC_CLEANUP_INTERVAL
               value: {{ .Values.cleanupInterval | quote }}
             - name: RENKU_SVC_WORKER_QUEUES
-              value: {{ .Values.workerQueues }}
+              value: {{ .Values.datasetsWorkerQueues }}
             - name: RENKU_SVC_CLEANUP_TTL_FILES
               value: {{ .Values.cleanupFilesTTL | quote }}
             - name: RENKU_SVC_CLEANUP_TTL_PROJECTS
               value: {{ .Values.cleanupProjectsTTL | quote }}
             - name: RQ_WORKER_LOG_LEVEL
               value: {{ .Values.logLevel }}
+          volumeMounts:
+            - name: shared-volume
+              mountPath: {{ .Values.cacheDirectory }}
+
+        - name: {{ .Chart.Name }}-management-workers
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["python"]
```

Comment:
you want to run this without gunicorn?

Reply:
It's a worker, no need for it. We should think of daemonizing it internally, but for now since we are running it on k8s, it should be fine.